runtime/doc/syntax.txt

```diff
@@ -1846,7 +1865,16 @@ Empty `*.m` files will automatically be presumed to be Matlab files unless you
 have the following in your vimrc: >
 
 	let filetype_m = "mma"
-
+<
+MEDIAWIKI					*ft-mediawiki-syntax*
+
+Be default, syntax highlighting includes basic HTML tags like style and
+headers |html.vim|. For strict Mediawiki syntax highlighting: >
+	let g:html_no_rendering = 1
+If HTML highlighting is desired, terminal-based text formatting such as bold
+and italic is possible by: >
+	let g:html_style_rendering = 1
+<
```

Comment:
```suggestion

MEDIAWIKI					*ft-mediawiki-syntax*

Be default, syntax highlighting includes basic HTML tags like style and
headers |html.vim|. For strict Mediawiki syntax highlighting: >

	let g:html_no_rendering = 1

If HTML highlighting is desired, terminal-based text formatting such as bold
and italic is possible by: >

	let g:html_style_rendering = 1
```

Reply:
I don't think we should accept broken syntax any longer which can mess up header detection. Our primary "record of truth" for valid vimdoc syntax is now our tree-sitter parser, not upstream's Pirates' Code.

Reply:
What messes up header detection? Upstream docs looks completely fine with treesitter.

Reply:
Yes, in this case. Other cases have _not_ been fine and I had to correct them manually to get the docs check to pass.